### PR TITLE
Team Page and Component

### PR DESCRIPTION
**Pull request by Feruzteame**

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+              <GatsbyImage className='w-44 h-44'
+              image={ image.image }
+              formats={ ['auto', 'webp'] }
+              alt={ image.alt } />
+            ) }
+            { title && (
+              <p className='
+                text-center 
+                font-normal text-sm
+              '>
```
Comment (matt-harding-2021) [inline src/components/Card.js]: p className='text-center font-normal text-sm'

File: src/cms/preview-templates/TeamPagePreview.js [hunk 2]
```diff
@@ -18,6 +18,21 @@ const TeamPagePreview = ({ entry, getAsset }) => {
...
           paragraph: data.heroPanel.paragraph
         }}
+        teamPanel={{
+          cards:{
+            image: {
+              image: getAsset(data.teamPanel.cards.image.image),
+              alt: data.teamPanel.cards.image.alt,
+            },
+            title: data.teamPanel.cards.image.title,
+            Subheading: data.teamPanel.cards.image.Subheading,
```
Comment (matt-harding-2021) [inline src/cms/preview-templates/TeamPagePreview.js]: S

File: src/cms/preview-templates/TeamPagePreview.js [hunk 3]
```diff
@@ -18,6 +18,21 @@ const TeamPagePreview = ({ entry, getAsset }) => {
...
+            linkedin: {
+              image: getAsset(data.teamPanel.cards.linkedin.image),
+              alt: data.teamPanel.teamCards.linkedin.alt,
+              link: data.teamPanel.teamCards.linkedin.link,
+            },
```
Comment (matt-harding-2021) [inline src/cms/preview-templates/TeamPagePreview.js]: only link needed, linkedin: data.teamPanel.teamCards.linkedin (string)

File: src/components/Card.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                { title }
+              </p>
+            ) }
+            { subheading && (
+              <p className='
+                text-center
+                font-light text-xs text-smallText
```
Comment (matt-harding-2021) [inline src/components/Card.js]: text-smallText, remove for now
p className='text-center font-normal text-sm'

File: src/components/Card.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+              <div className='
+                flex 
+                justify-center
+                w-full
+              '>
+                <a href={ linkedin.link } target='_blank'>
+                  <img className='h-8 w-8'
+                  src={ linkedin.image }
+                  alt={ linkedin.alt } />
+                </a>
```
Comment (matt-harding-2021) [inline src/components/Card.js]: rel="noopener noreferrer"

File: src/components/Card.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+                flex 
+                justify-center
+                w-full
+              '>
+                <a href={ linkedin.link } target='_blank'>
+                  <GatsbyImage className='h-12 w-12'
+                  image={ linkedin.image }
+                  formats={ ['auto', 'webp'] }
+                  alt={ linkedin.alt } />
+                </a>
```
Comment (matt-harding-2021) [inline src/components/Card.js]: noreferrer

File: src/components/TeamPanel.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+                      image : getImage(card.image.image) || card.image.image,
+                      alt: card.image.alt
+                    } }
+                    title={ card.title }
+                    subheading={ card.subheading }
+                    linkedin={ {
+                      image: getImage(card.linkedin.image) || card.linkedin.image,
+                      alt: card.linkedin.alt,
+                      link: card.linkedin.link
+                     } } />
```
Comment (matt-harding-2021) [inline src/components/TeamPanel.js]: Naming 'Card' is too generic, TeamPageCard, or *ProfileCard, TeamCard

File: src/pages/team/index.md [hunk 8]
```diff
@@ -13,6 +13,35 @@ heroPanel:
...
     Driven by the desire to use tech to bring transparency and reduce operational tasks, Finlight was born.
   button:
+teamPanel:
+  cards:
+    - image:
+        image: /img/brussels.png
+        alt: img1
+      title: name
+      subheading: profession
+      linkedin:
```
Comment (matt-harding-2021) [inline src/pages/team/index.md]: linkedin: https://finlight.com/

File: tailwind.config.js [hunk 9]
```diff
@@ -17,6 +17,7 @@ module.exports = {
       colors: {
         'primary': '#E7B224',
         'secondary': '#3273DC',
+        'smallText' : '#242424'
```
Comment (matt-harding-2021) [inline tailwind.config.js]: naming should relate to the color: delete, and we'll do it later

**Review by matt-harding-2021 — COMMENTED** [review]
comments

File: src/templates/team-page.js [hunk 10]
```diff
@@ -93,6 +101,21 @@ export const teamPageQuery = graphql`
           titleHighlight
           paragraph
         }
+        teamPanel {
+          cards {
+            image {
```
Comment (matt-harding-2021) [inline src/templates/team-page.js]: cardsImage

**Review by ClaudiuCornea — APPROVED** [review]

**Review by ClaudiuCornea — APPROVED** [review]

**Pull request merged**